### PR TITLE
Chinese localization

### LINKED ISSUE #21: Chinese localization
I have created a Chinese localization file for this mod and hope it can be integrated into the official version.
[zh_cn.json](https://github.com/user-attachments/files/16618478/zh_cn.json)

### PR DESCRIPTION
**Pull request by Baichuan-qiao**
add "zh_cn.json"

### CODE REVIEW

File: src/main/resources/assets/flightassistant/lang/zh_cn.json
Comment (Octol1ttle) [inline src/main/resources/assets/flightassistant/lang/zh_cn.json:1]: I no longer use JSON for language files. I also noticed there are a few strings that are missing or out of date, please check out the latest language file at https://github.com/Octol1ttle/FlightAssistant/blob/dev/src/main/resources/assets/flightassistant/lang/en_us.yml and update your pull request accordingly.

**Review by Octol1ttle — CHANGES_REQUESTED** [review]

File: src/main/resources/assets/flightassistant/lang/zh_cn.json
Comment (Baichuan-qiao) [reply src/main/resources/assets/flightassistant/lang/zh_cn.json:1]: I will complete it soon.

Comment (Baichuan-qiao) [reply src/main/resources/assets/flightassistant/lang/zh_cn.json:1]: I have completed it.

**Comment by Baichuan-qiao** [comment]
I have revised the original unreasonable translation and translated the new features.

**Review by Octol1ttle — APPROVED** [review]

**Pull request merged**